src/www/ui/api/documentation/openapi.yaml

```
@@ -797,6 +797,79 @@ paths:
                           "fossy"        
         default:
           $ref: '#/components/responses/defaultResponse'
+  /users/tokens:
+    post:
+      operationId: createRestApiToken
+      tags:
+        - User
+        - Admin
+      summary: Create a new REST API Token
+      description: >
+        Create a new REST API Token
+      requestBody:
+        required: true
+        content:
+          application/x-www-form-urlencoded:
+            schema:
+              type: object
+              properties:
+                token_name:
+                  type: string
+                token_scope:
+                  type: string
+                token_expire:
+                  type: string
+              required:
+                - token_name
+                - token_scope
+                - token_expire
```

Use the existing data model instead of creating a new one.
https://github.com/fossology/fossology/blob/5eaca363ca51c212038939acc98c24672b6392e0/src/www/ui/api/documentation/openapi.yaml#L1791
```suggestion
            schema:
              $ref: '#/components/schemas/TokenRequest'
```

---

Did the required changes. PR is updated now!